crates/driver/src/infra/config/file/mod.rs

```diff
@@ -89,6 +89,15 @@ enum Mempool {
         /// Flashbots above regular gas price estimation.
         #[serde(default = "default_max_additional_flashbots_tip")]
         max_additional_tip: f64,
+        /// Configures whether the submission logic is allowed to assume the
+        /// submission nodes implement soft cancellations. With soft
+        /// cancellations you don't have to submit a no-op transaction to
+        /// replace another transaction in the mempool. Instead you can just
```

Comment:
```suggestion
        /// cancellations the driver doesn't have to submit a no-op transaction to
        /// replace another transaction in the mempool. Instead it can just
```